README.md

```diff
@@ -33,6 +35,19 @@ use Symfony\Component\Routing\RouteCollectionBuilder;
     // ...
 ```
 
+Then make sure your `serverless.yaml` contains at least the following include/exclude rules
+
+```yaml
+# ...
+package:
+    exclude:
+        - var/**
+    include:
+        - var/cache/prod/**
+        - var/build/prod/**
```

Comment:
I know most of the time we deploy via CI, but I'm thinking about a use case that goes like this:

- I want to try out Bref and Lambda
- I create a new Symfony application
- I install the Bref bridge
- I run the app locally (dev cache generated)
- I deploy to Lambda:
  - generate prod cache
  - `serverless deploy`

I'm wondering about 2 things:

1. is the dev cache a problem on Lambda? (I'm guessing that no, it's not deployed, but just to be sure)
1. is my local dev environment still working?

I _think_ there is no problem, but I'd rather ask to people who know more ^^

Reply:
dev would not be a problem I tested to deploy from local and it worked great even with a var/cache/dev & var/build/dev
1. is the dev cache a problem on Lambda? :arrow_right: Not deployed, not conflicting if warmed locally but not deployed
2. is my local dev environment still working? :arrow_right: yes it's working seemlessly

And I'm also testing with a CI (GH actions) and it's working great too. 

I've tested this approach on [symfony/demo](https://github.com/symfony/demo) which is a fully featured app : 
* Doctrine
* Postgres database
* Users with sessions
* Twig templates
* Webpack encore for the assets

Note that is a fully WIP PR and want to make sure this will work well with hopefully any setup

Reply:
Beautiful, thanks!